doc/source/intro.rst

```diff
@@ -75,14 +75,26 @@ Tested Environment
 ******************
 
 Currently, DCE only supports Linux-based operating systems.
-DCE has been tested on the following distributions:
+The current version of DCE has been tested only on the following distribution:
 
-- Ubuntu 16.04 and equivalents (gcc-5.4, libc-2.23)
+- Ubuntu 16.04 (gcc-5.4, libc-2.23)
 
-but you can try on others (e.g., CentOS, RHEL).  However, unfortunately
-only a narrow range of older Linux systems work with DCE (gcc-5-based
-systems such as Ubuntu 16.04).
+Newer Linux distributions (based on newer libc) require different support and
+are not presently supported but are under development.
 
 If you find that you can run on another distribution, please let the
 :ref:`maintainers` know.  Also, if you would like to work on getting DCE to 
 work on newer systems, please let the :ref:`maintainers` know.
+
+Roadmap
+*******
+
+The DCE roadmap is maintained `here <https://github.com/direct-code-execution/ns-3-dce/wiki/Roadmap>`_.
+
+DCE version 1.11 is the last release for Ubuntu 16.04, and is compatible with 
+the `ns-3.34 release <https://www.nsnam.org/releases/ns-3-34/>`_.  The (optional) Linux kernel version
+supported is 4.4 `(net-next-nuse-4.4.0) <https://github.com/libos-nuse/net-next-nuse/releases/tag/libos-v4.4>`_.
+
+DCE version 1.12 is planned for use on Ubuntu 20.04 but with minimal changes from release 1.11.
```

Comment:
this will get outdated, better to leave it on the wiki that is easier to change. Other than that feel free to merge.